woocommerce/class-sv-wc-helper.php

```diff
@@ -1041,16 +1041,32 @@ public static function is_enhanced_admin_screen() {
 	 *
 	 * @since 5.10.6
 	 *
+	 * @deprecated with no alternatives
+	 *
 	 * @return bool
 	 */
-	public static function is_wc_navigation_enabled() {
-
-		return
-			is_callable( [ \Automattic\WooCommerce\Admin\Features\Navigation\Screen::class, 'register_post_type' ] ) &&
-			is_callable( [ \Automattic\WooCommerce\Admin\Features\Navigation\Menu::class, 'add_plugin_item' ] ) &&
-			is_callable( [ \Automattic\WooCommerce\Admin\Features\Navigation\Menu::class, 'add_plugin_category' ] ) &&
-			is_callable( [ \Automattic\WooCommerce\Admin\Features\Features::class, 'is_enabled' ] ) &&
-			\Automattic\WooCommerce\Admin\Features\Features::is_enabled( 'navigation' );
+	public static function is_wc_navigation_enabled() : bool
+	{
+		if (SV_WC_Plugin_Compatibility::is_wc_version_gte('9.3')) {
+			self::enhancedNavigationDeprecationNotice();
+			return false;
+		}
+
+		return self::isEnhancedNavigationFeatureEnabled();
+	}
+
+	protected static function isEnhancedNavigationFeatureEnabled() : bool
```

Comment:
Missing a docblock here for the new method.
```suggestion
	/**
	 * Determines whether Woo's Enhanced Eavigation feature is enabled.
	 *
	 * @since 5.15.1
	 *
	 * @return bool
	 */	 
	protected static function isEnhancedNavigationFeatureEnabled() : bool
```